charts/pxc-operator/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v2
-appVersion: 1.10.0
+appVersion: 1.10.1
```

Comment:
Please leave this at 1.10.0 and just bump the chart version to 1.10.1 because the operator version stays the same.